test/WebJobs.Script.Tests/Diagnostics/FunctionInstanceLoggerTests.cs

```diff
@@ -30,24 +30,20 @@ public void LogInvocationMetrics_EmitsExpectedEvents()
                 Name = "TestFunction"
             };
             metadata.Bindings.Add(new BindingMetadata { Type = "httpTrigger" });
-            metadata.Bindings.Add(new BindingMetadata { Type = "blob", Direction = BindingDirection.In });
```

Comment:
I think you should leave the existing test inputs as is, since this represents a valid user scenario (in and out bindings of the same type). Instead just update the log verification.

Reply:
Test updated in the latest iteration.